### PR TITLE
feat: add webhook types

### PR DESCRIPTION
**Pull request by vitords**
This PR introduces the Monta webhook type structs to be used with the GET, UPDATE, DELETE and RETRIEVE webhook endppoints.

### CODE REVIEW

File: webhookentitytype.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package monta
+
+// WebhookEtityType represents the type of a webhook entity.
+type WebhookEntityType string
+
+// Known [WebhookEtityType] values.
+const (
```
Comment (CodingBoulderingRepeat) [inline webhookentitytype.go:7]: They generally support more than this right? But these are the only ones that are relevant right now?

File: webhook.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package monta
+
+// WebhookRequest is a webhook request from Monta.
+type WebhookRequest struct {
+	// List of webhook entries delivered in a batch.
+	Entries []WebhookEntry `json:"entries"`
+
+	// Number of pending events left for delivery.
```
Comment (CodingBoulderingRepeat) [inline webhook.go:8]: Does this mean the ones left not considering the entries delivered in this batch?

File: webhook.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	Pending int64 `json:"pending"`
+
+	// Timestamp of the request (milliseconds from the epoch of 1970-01-01T00:00:00Z).
+	Timestamp UnixTimestamp `json:"timestamp"`
+}
+
+// WebhookEntry is an entry of the webhook request.
+type WebhookEntry struct {
+	// Type of the entity.
+	EntityType string `json:"entityType"`
```
Comment (CodingBoulderingRepeat) [inline webhook.go:18]: Use `WebhookEntityType` here?
```suggestion
	EntityType WebhookEntityType `json:"entityType"`
```

**Review by CodingBoulderingRepeat — APPROVED** [review]
Nice! My comments are not blocking.

File: webhook.go (hunk 2 above)
Comment (vitords) [reply webhook.go:8]: That's what I understood from the docs.

File: webhookentitytype.go (hunk 1 above)
Comment (vitords) [reply webhookentitytype.go:7]: There is and they will add more from time to time. These were the ones available when this was implemented.

**Pull request merged**